Source/RapyutaSimulationPlugins/Private/Core/RRGameMode.cpp

```diff
@@ -138,6 +138,9 @@ void ARRGameMode::StartPlay()
     // 2- START SIM
     StartSim();
 
+    // 3- START ROS2 Nodes
+    InitSim();
+
```

Comment:
Is that `InitSim()` now already called in `ARRROS2GameMode::StartPlay()`, which happens after all resources loading?

Reply:
StartPlay is overwridden without calling Super::StartPlay() in RRGameMode

Reply:
Yeah, it is still called [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Private/Core/RRGameMode.cpp#L203) as in a deferred style after all resources have been loaded.